### PR TITLE
Revert changes that are enhancement.

### PR DESCRIPTION
**Pull request by nubnubyas**
These are the changes reverted:

1. Revert the message format for a person's details that are displayed. (Original: #230, Commit: https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/230/commits/442263687de1fafc8e2fa3563938d2f75d11d52d)
2. Revert changes for Email validation. (Original: #230, Commit: https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/230/commits/3065c5c4521cc38b8b67618e9bddb1540f39ed6e)
3. Revert changes that make `Id` always capitalized (Original: #204, Commit: https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/204/commits/5c9b61709088f530b8256402e18c0ab4c85a1361)
4. Revert changes for message displayed for FindCommand when filteredList is empty. (Original: #204, Commit: https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/204/commits/c8974107794043d004b43d4b6ffffebbace2ead3)
5. Revert changes for Appointment to be case insensitive. (Original: #204, Commit: https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/204/commits/4c53aacd450446766d5922a3200fe45698bfc164)
6. Revert changes make to message displayed after executing `find`. (Original: #204, Commit: https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/204/commits/c8974107794043d004b43d4b6ffffebbace2ead3)

### CODE REVIEW

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -57,14 +54,10 @@ public static String format(Person person) {
                 .append(person.getAddress())
                 .append(";\n")
                 .append("Appointment: ")
-                .append(person.getAppointment().map(Appointment::toString).orElse("N/A"))
```
Comment (WZWren) [inline src/main/java/seedu/address/logic/Messages.java:60]: I liked the `.map()` change, it improved code quality a lot. May I suggest that you keep the `null` but use `map`?

Example:
```
person.getAppointment().map(Appointment::toString).orElse(null)
```

**Review by WZWren — COMMENTED** [review]
It is sad that we have to revert these changes, but we have to do this for the deliverables. However, I think there's a good change that we are reverting, that only improves code quality. Consider keeping that one.

File: src/main/java/seedu/address/logic/Messages.java (hunk 1 above)
Comment (nubnubyas) [reply src/main/java/seedu/address/logic/Messages.java:60]: Good suggestion, I will keep the use of map.

**Comment by WZWren** [comment]
All looks good, merging.

**Pull request merged**